styles/style.css

```diff
@@ -61,6 +59,33 @@ body {
 }
 
 .card-thumbnail {
-  max-height: 250px;
-  overflow: hidden;
+    width: 100%;
+    height: 17vw;
+    border-bottom: 2px solid rgba(0, 0, 0, .26);
 }
+
+.card-thumbnail img {
```

Comment:
I would say maybe do some adjustments to the image sizes so they fit in the thumbnail box, some are a bit zoomed in. Obviously doesn't hurt the functionality though!